main.cpp

```diff
@@ -1,475 +1,581 @@
+*/
 
-Move them below this block comment and put them in numerical order 
-(1 - 10).
+struct Medic
+{
+    //qualifications (std::string)
+    std::string qualifications;
+    //name (std::string)
+    std::string name = "The FA Level 4 Emergency Medical Management in Football";
+    //salary (double)
+    double salary = 178909.99;
+    //explayer (bool)
+    bool isExPlayer = true;
+    //IT skills (bool)
+    bool hasITSkills = true;
+
+    //give steriod injection
+    void giveSteriodInjection(Player player, std::string bodyPart, bool servePain);
+    //massage treatment
+    void giveMassageTreatment(Player player, std::string therapyTargetArea);
+    //make fitness assessment
+    float getFitnessAssessmentScore(int restingBPM, int maxBPM, float bloodPressure); //returns value of a players fitness assessment
+};
 
-The next thing that will happen is that I will review the 
-pseudo-code that you have written.
-Send me a DM to review your pull request when the project is ready for review.
+/*
+Thing 9) Stadium
+5 properties:
+    1) number of seats (int)
+    2) pitch area (float)
+    3) number of toilets (int)
+    4) training pitch (bool)
+    5) construction cost (double)
+3 things it can do:
+    1) host home game
+    2) covid vaccination centre
+    3) require maintance
 */
 
+struct Stadium
+{
+    //number of seats (int)
+    int numSeats = 62850;
+    //pitch area (float)
+    float pitchAreaInSqMetres = 7123.89f;
+    //number of toilets (int)
+    int numToilets = 66;
+    //training pitch (bool)
+    bool trainingPitch = true;
+    //construction cost (double)
+    double constructionCost = 1200500700.99;
+
+    //host home game
+    void homeGame(std::string nameVistingTeam);
+    //covid vaccination centre
+    bool covidVaccinationCentre(int numberOfUnvaccinated, float RNumber); //returns wheteher stadium is required to be vaccination centre
+    //require maintance
+    void maintance(bool brokenToilet);
+};
+
 /*
- MAKE SURE YOU ARE NOT ON THE MASTER BRANCH
+Thing 10) Sports Team
+5 properties:
+    1) player
+    2) coach
+    3) fan
+    4) medic
+    5) stadium
+3 things it can do:
+    1) play home game
+    2) play away game
+    3) win championship
+*/
+struct TeamCaptain //to prevent compile errors
+{
+    std::string name = "David Davison";
+};
+struct NumberOneFan //to prevent compile errors
+{
+    std::string name = "William Williamson";
+};
 
- Commit your changes by clicking on the Source Control panel on the left, entering a message, and click [Commit and push].
- 
- If you didn't already: 
-    Make a pull request after you make your first commit
-    pin the pull request link and this repl.it link to our DM thread in a single message.
 
- send me a DM to review your pull request when the project is ready for review.
+struct SportsTeam
+{
+    //player
+    Player player;
+    //coach
+    Coach coach;
+    //fan
+    NumberOneFan numberOneFan;
+    //medic
+    TeamCaptain TeamCaptain;
```

Comment:
Use an instance of `Medic` instead.  
You already have the class for it declared above.
- [ ] revise